### ISSUE TITLE
Crashed (Baseline Unet model training for Covid-19 sementation challenge)

### ISSUE THREAD

**Issue by rashidch** [Project-MONAI/tutorials#69]
(pytorch) rasho@rasho-WS-E500-G5-WS690T:~/covid-19_3D_Segmentation$ python run_net.py train --data_folder "COVID-19-20_v2/Train" --model_folder "runs"
MONAI version: 0.3.0+81.g62b0bbb
Python version: 3.7.9 (default, Aug 31 2020, 12:42:55)  [GCC 7.3.0]
OS version: Linux (5.4.0-53-generic)
Numpy version: 1.19.2
Pytorch version: 1.7.0
MONAI flags: HAS_EXT = False, USE_COMPILED = False

Optional dependencies:
Pytorch Ignite version: 0.4.2
Nibabel version: 3.2.0
scikit-image version: NOT INSTALLED or UNKNOWN VERSION.
Pillow version: 8.0.1
Tensorboard version: NOT INSTALLED or UNKNOWN VERSION.
gdown version: NOT INSTALLED or UNKNOWN VERSION.
TorchVision version: 0.8.1
ITK version: NOT INSTALLED or UNKNOWN VERSION.
tqdm version: 4.52.0

For details about installing the optional dependencies, please visit:
    https://docs.monai.io/en/latest/installation.html#installing-the-recommended-dependencies

INFO:root:training: image/label (199) folder: COVID-19-20_v2/Train
INFO:root:training: train 160 val 39, folder: COVID-19-20_v2/Train
INFO:root:batch size 2
Load and cache transformed data: 100%|█████████████████████████████████████████████████████████████████████| 160/160 [04:13<00:00,  1.58s/it]
Load and cache transformed data: 100%|███████████████████████████████████████████████████████████████████████| 39/39 [00:58<00:00,  1.51s/it]
BasicUNet features: (32, 32, 64, 128, 256, 32).
INFO:root:epochs 500, lr 0.0001, momentum 0.95
INFO:ignite.engine.engine.SupervisedTrainer:Engine run resuming from iteration 0, epoch 0 until 500 epochs
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 1/80 -- train_loss: 1.4053 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 2/80 -- train_loss: 1.3833 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 3/80 -- train_loss: 1.3598 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 4/80 -- train_loss: 1.3268 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 5/80 -- train_loss: 1.3438 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 6/80 -- train_loss: 1.3146 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 7/80 -- train_loss: 1.3164 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 8/80 -- train_loss: 1.3118 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 9/80 -- train_loss: 1.2970 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 10/80 -- train_loss: 1.2957 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 11/80 -- train_loss: 1.2779 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 12/80 -- train_loss: 1.2499 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 13/80 -- train_loss: 1.2641 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 14/80 -- train_loss: 1.2634 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 15/80 -- train_loss: 1.2439 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 16/80 -- train_loss: 1.2206 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 17/80 -- train_loss: 1.2209 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 18/80 -- train_loss: 1.2143 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 19/80 -- train_loss: 1.1976 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 20/80 -- train_loss: 1.1950 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 21/80 -- train_loss: 1.1833 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 22/80 -- train_loss: 1.1747 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 23/80 -- train_loss: 1.1739 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 24/80 -- train_loss: 1.1676 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 25/80 -- train_loss: 1.1586 
INFO:ignite.engine.engine.SupervisedTrainer:Epoch: 1/500, Iter: 26/80 -- train_loss: 1.1585 
Killed

**Comment by rashidch** [comment]
Model training crashed after few iterations without showing any error. Can anyone tell the reason why it is crashed?

**Comment by ericspod** [comment]
I can't see anything in the printout that would indicate why this crash occurred. What hardware are you using? What software libraries other than those listed here? What distro? If the job was killed without any error output it could be driver or hardware related.
